chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -80,6 +80,25 @@
 
 when "db_shutdown"
 
+  # Stopping postgresql service in HA and non-HA
+  if File.exist?("/usr/sbin/crm")
+  bash "Stop postgresql" do
```

Comment:
Style/IndentationWidth: Use 2 (not 0) spaces for indentation. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)